src/forms/buttons/button.js

```diff
@@ -53,17 +52,29 @@ const propTypes = {
 }
 
 const defaultProps = {
-  style: 'primary',
+  priority: 'primary',
```

Comment:
[Emphasis](https://semantic-ui.com/elements/button.html#emphasis) or [Color](https://material-ui.com/components/buttons/#group-sizes-and-colors)?

Reply:
This might be the only time I say this, but I might go with bootstrap on this one https://react-bootstrap.github.io/components/buttons/#examples